src/Components/Footer.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { NavMenu } from './NavMenu';
+import { ForComletedTodo } from '../types/enumFilter';
+
+type Props = {
+  todos: Todo[];
+  condition: ForComletedTodo;
+  setCondition: (condition: ForComletedTodo) => void;
+  handleClearCompletedTodos: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  condition,
+  setCondition,
+  handleClearCompletedTodos,
+}) => {
+  const todosNoComleted = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${todosNoComleted} items${todosNoComleted !== 1 ? '' : ''} left`}
```

Comment:
As I understand it, it should be like that here?
```suggestion
        {`${todosNoComleted} item${todosNoComleted === 1 ? '' : 's'} left`}
```